src/open_clip/model.py

```diff
@@ -29,6 +29,7 @@ class CLIPVisionCfg:
     patch_size: int = 16
     image_size: Union[Tuple[int, int], int] = 224
     ls_init_value: Optional[float] = None  # layer scale initial value
+    patch_dropout: float = 0.  # what percentage of patches to dropout during training (not inference) - 0.5 to 0.75 recommended in the paper for optimal results
```

Comment:
this needs to be exposed through params.py

Reply:
ah no I see, configured that way it would need to be in one of the model config json files

Reply:
it should default to 0. even if `patch_dropout` is omitted from all the json files?

Reply:
yes that's right

Reply:
I'm trying to think how we want to use that. Duplicating all the config files to be able to use it seems a little weird
maybe it should be a param in params.py and then we have a `model.patch_dropout = params.patch_dropout` somewhere

but let's not overthink it, I'll try it in the current state first

Reply:
Yup, agreed! Let's see how effective it is in the Open clip setting

If it becomes a necessity, then we can add it to all the configs with the optimal dropout value